api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -1044,6 +1044,31 @@ paths:
       security:
         - OAuth2:
             - SIG/ALL
+  /signals/v1/public/questions/:
+    get:
+      description: Retrieve a list of questions in the database. Experimental, the content of the response can still be changed
+      parameters:
+        - name: main_slug
+          in: query
+          description: Slug that identifies the main category.
```

Comment:
Maybe add that this filter only returns questions associated a main category. Not the questions associated with sub categories of this main category. Specify `sub_slug` in combination with `main_slug` to get a sub category's questions.